src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  sourceString.split(';').forEach((style) => {
```

Comment:
You can try to use reduce. In this case, you don't need initialize stylesObject on 10 line (optional)

Reply:
Edited) Thank you!